packages/x-tailwindcss/demo/src/components/xds-suggestion.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <XdsBaseShowcase
+    #default="{ cssClass, section, copyCssClassesToClipboard, removeClassPrefix }"
+    title="Suggestion"
+    :sections="sections"
+  >
+    <div
+      :key="cssClass"
+      @click="copyCssClassesToClipboard"
+      :class="cssClass"
+      aria-hidden="true"
+      title="Click me to copy CSS classes"
+    >
+      <CuratedIcon class="x-icon" />
+      <div v-if="section === 'Matching'">
+        <span class="x-suggestion-matching-part">su</span>
+        <span>ggestion</span>
+      </div>
+      <span v-else>{{ removeClassPrefix(cssClass, base) }} suggestion</span>
+    </div>
+  </XdsBaseShowcase>
+</template>
+
+<script lang="ts">
+  import { Vue, Component, Prop } from 'vue-property-decorator';
+  import { ShowcaseSections } from '../types/types';
+  import { addParentClasses } from '../utils';
+  import CuratedIcon from './icons/curated.vue';
+  import XdsBaseShowcase from './xds-base-showcase.vue';
+
+  @Component({
+    components: {
+      CuratedIcon,
+      XdsBaseShowcase
+    }
+  })
+  export default class XdsSuggestion extends Vue {
+    @Prop({ default: () => 'x-suggestion' })
+    public base!: string;
+
+    @Prop({
+      default: () => ['x-suggestion-md', 'x-suggestion-lg']
+    })
+    public sizes!: string[];
+
+    @Prop({
+      default: () => [
+        '',
```

Comment:
why '' as value?

Reply:
I added it so that an example with no color applied is displayed for the other sections whose examples iterate over the colors.